ethers-core/src/types/transaction/eip2718.rs

```diff
@@ -650,6 +763,20 @@ impl TypedTransaction {
             },
             #[cfg(feature = "optimism")]
             OptimismDeposited(tx) => Eip2930TransactionRequest { tx: tx.tx, access_list },
+            #[cfg(feature = "kroma")]
+            KromaDeposited(_) => Eip2930TransactionRequest {
+                tx: TransactionRequest {
+                    from: self.from().copied(),
+                    to: self.to().cloned(),
+                    nonce: self.nonce().copied(),
+                    value: self.value().copied(),
+                    gas: self.gas().copied(),
+                    gas_price: self.gas_price(),
+                    chain_id: self.chain_id(),
+                    data: self.data().cloned(),
```

Comment:
Why it's different from optimism one?

Reply:
@dongchangYoo I just copied the above code starting from [this line](https://github.com/kroma-network/ethers-rs/pull/1/files#diff-c57e9a06ef7b5f55c5498a79e30dfd71f1e395b11c58a50d340a6276bf0dec65R742).

Reply:
@chokobole It's because Our `TypedTransaction::KromaDeposited` is different from `TypedTransaction::OptimismDeposited`. The reason why I didn't follow the same to `OptimismDeposited` is because I thought there were some unnecessary fields in legacy txs for `KromaDeposited`.